package.json

```diff
@@ -40,7 +40,7 @@
   },
   "dependencies": {
     "@juggle/resize-observer": "^3.2.0",
-    "@webex/component-adapter-interfaces": "^1.13.0",
+    "@webex/component-adapter-interfaces": "^1.16.0",
```

Comment:
please remove this, it is handled by another PR https://github.com/webex/components/pull/204